examples/features/encryption/ALTS/client/main.go

```diff
@@ -0,0 +1,63 @@
+/*
+ *
+ * Copyright 2018 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/alts"
+	pb "google.golang.org/grpc/examples/helloworld/helloworld"
+)
+
+const (
+	address = "localhost:50051"
```

Comment:
Let's make `address` and `port` on the server side configurable.

https://github.com/grpc/grpc-go/blob/master/examples/features/metadata/client/main.go#L35
https://github.com/grpc/grpc-go/blob/master/examples/features/metadata/server/main.go#L39